python/lib/metrics/dmod/test/uniform_metrics_and_thresholds.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python
+import typing
+import os
+
+from argparse import ArgumentParser
+
+import pandas
+
+from dmod.metrics.threshold import Threshold
+import dmod.metrics.scoring as scoring
+import dmod.metrics.metric as metrics
```

Comment:
A few more imports to adjust like those mentioned above.

Reply:
Just removed the file - it was a manual script meant to be run for diagnostic/development purposes, not as a unit test. The imports don't work as relative in this scenario because: `ImportError: attempted relative import with no known parent package`. The properties that _were_ tested are also in `test_scoring`.